Utils/Dataflow/030_PDFAnalyzer/xmltable.py

```diff
@@ -138,8 +138,8 @@ def merge(self, line):
 def row_centery(row):
     """ Calculate average y-center in a row. """
     c = 0
-    for l in row:
-        c += l.center[1]
+    for line in row:
```

Comment:
May be "cell" will be better than "line" here?

Reply:
@Evildoor,
please consider.

Reply:
I don't think so. We had our share of trouble with naming elements of tables in this code, and for now settled on table consisting of rows, and row consisting of lines. This function correlates with the code by using these names. 

If "line" is a bad name, then it should be changed in the whole code, not just here. "Cell" does not sounds right to me because our lines do not correspond to cells in original tables in PDF documents.

Reply:
If the rest of the code is to be fixed with respect to the naming -- it should be fixed.  If it shouldn't -- OK, I was wrong and I apologize.  If you'd show me what you mean by "our lines do not correspond to cells in original table in PDF documents" (simple link to the PDF document and few words explaining where to look are enough) -- I'd be rather grateful.

Reply:
Will answer via email.